src/utils/update-config.js

```diff
@@ -29,7 +29,13 @@ const parseXML = function(xmlPath) {
 };
 
 const saveXML = function(json, xmlPath) {
-  const builder = new xml2js.Builder();
+  const builder = new xml2js.Builder({
+    renderOpts: {
+      'pretty': true,
+      'indent': '    ',
+      'newline': '\n'
```

Comment:
Can't this be CRLF on windows? You might need to detect the line ending so it doesn't make the whole file show up as changed.